src/App.tsx

```diff
@@ -1,26 +1,32 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import styled from 'styled-components'
 
-function App() {
+import CanvasContextProvider from './contexts/CanvasContextProvider';
+import ImagePane from './components/ImagePane'
+import DataPane from './components/DataPane'
+
+
+type AppProps = {
+  className?: string
+}
+
+const AppJSX = ({className}: AppProps) => {
```

Comment:
The -JSX suffix is a personal invention to mark components that are not styled yet. It may not be clear at first sight, but I haven't come up with any better one yet. If you have any suggestion, please let me know!

I didn't know about Hungarian notation until now, and I think it's quite cool! Thanks for sharing with me!